src/schema/v2/artists/curatedTrending.ts

```diff
@@ -1,11 +1,31 @@
 import { GraphQLFieldConfig } from "graphql"
 import type { ResolverContext } from "types/graphql"
 import { connectionArgs, connectionFromArray } from "graphql-relay"
-
+import Artist from "schema/v2/artist"
 import { dailyShuffle } from "lib/shuffle"
 import { ArtworksAggregation } from "../aggregations/filter_artworks_aggregation"
 import { artistConnection } from "../artist"
 
+export const getCuratedArtists = async (context): Promise<[typeof Artist]> => {
+  const { artistsLoader, filterArtworksLoader, defaultTimezone } = context
+  const artworks = await filterArtworksLoader({
+    size: 0,
+    marketing_collection_id: "trending-this-week",
```

Comment:
**thought (non-blocking):** Reminds me that this is a bit of a weak link. If anything happens to this `trending-this-week` collection, there is an increasingly wide blast radius. Would be good to add some kind of admin-facing note in the admin UI for this, somehow.